src/ui/SmartSnippet/SmartSnippetSuggestions.ts

```diff
@@ -35,15 +40,34 @@ export class SmartSnippetSuggestions extends Component {
     });
   };
 
+  /**
+   * The options for the SmartSnippetSuggestions
+   * @componentOptions
+   */
+  static options: ISmartSnippetSuggestionsOptions = {
+    /**
+     * Specify if the SmartSnippetSuggestion snippet should be displayed inside an iframe or not.
+     *
+     * Use this option in specific cases where your environment has limitations around iframe usage.
+     *
+     * **Examples:**
+     *
+     * ```html
+     * <div class='CoveoSmartSnippetSuggestions' data-without-shadow='true'></div>
```

Comment:
Missing \`\`\` here too.
```suggestion
     * <div class='CoveoSmartSnippetSuggestions' data-without-shadow='true'></div>
     \`\`\`
```